helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -66,6 +66,8 @@ global:
       clientId: ""
       # global.hxpr.admin.clientId -- HxP Repository admin client secret
       clientSecret: ""
+    # global.hxpr.force-single-repository-url -- "when true, the repository URL is not based on the target environment, but forced to be global.hxpr.url
+    force-single-repository-url: false
```

Comment:
this should be `force` to match where it is used, should we keep `force-single-repository-url` better match other vars and use CamelCase `forceSingleRepositoryUrl`